### DISCUSSION TITLE
Weather System

### DISCUSSION THREAD

**Discussion by stevew00dy** [sunflower-land/sunflower-land#1132] [Ideas]
Posted from Wolfie on Discord

If stamina appears in the form of water in the expansion of the lands, and if weather is added to the game, then I have a little idea. What if during the rain there is a boost for all players, which increases the growth rate of plants by any %. In addition to all this, a bucket of water can also be filled with some small part of it. This will make it easier (if this relief is of course needed) for players to earn more sfl. Let the rain be a phenomenon that occurs spontaneously, for example, 3 times a week for 4 hours. This will not really ruin the economy, but not everyone will be able to take advantage of it, because for example, you plant carrots and wait for an hour, then you forget about the game for a hour, but because of the rain, carrots will grow for example one and a half times faster. Only after an hour you will come on farm and realize that rain is falling and in fact you could have come in earlier, harvested and planted something that grows longer. You can add a different % boost to each of the plant species separately. For example, you can increase wheat growth by 2 times by 4 hours, but for example carrots are grows only 50% faster, and potatoes are even 15% (I want people to have motivation to plant crops that grow the longer than others. The longer the vegetable grows, the stronger the boost will be during the rain).

**Comment by Flycry2021** [comment]
That's a good idea.

**Reply by Yougottabekiddingme123** [reply]
As Steve mentioned in the video. The weather system should not be regular and it should not be one way (only positive) due to rain. It should be used to manage the economy and slow harvesting when there is a boom and to Speed things up when there is a shortage of Sfl. 

Managing an economy is not easy and this will help to keep things balanced.
